.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -0,0 +1,109 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none
+pr: none
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily Build
+  branches:
+    include:
+    - 202012
+  always: true
+
+pool: sonicbld
+
+stages:
+- stage: Build
+  jobs:
+  - template: azure-pipelines-build.yml
+    parameters:
+      buildSlave: y
+- stage: UpgradeVersions
+  jobs:
+  - job: UpgradeVersions
+    pool:
+      vmImage: 'ubuntu-latest'
```

Comment:
suggest ubuntu-20.04 so that we can control it, otherwise, if ubuntu upgrade to 21.xx, then out build might break.

Reply:
Thanks for your comment, using the fix version 20.04 instead.